.github/workflows/ci.yml

```diff
@@ -10,28 +10,39 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  self-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: pnpm/action-setup@v4
```

Comment:
@NullVoxPopuli can/should we explicitly set the version? just thinking that it won't drift some day to v10 once it released plus it's explicit

Reply:
it uses the `packageManager` field

Reply:
oh cool! TIL